docs.html

```diff
@@ -0,0 +1,552 @@
+                    <div class="col">2 of 2</div>
+                </div>
+            </div>
+            <p>Three Items in a grid</p>
+            <div class="grid-container-3">
+                <div class="row">
+                    <div class="col">1 of 3</div>
+                    <div class="col">2 of 3</div>
+                    <div class="col">3 of 3</div>
+                </div>
+            </div>
+        </div>
+        <h4>Code Snippets</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+        &ltdiv class="grid-container-2"&gt
+            &ltdiv class="row"&gt
+                &ltdiv class="col"&gt1 of 2&lt/div&gt
+                &ltdiv class="col"&gt2 of 2&lt/div&gt
+            &lt/div&gt
+        &lt/div&gt
+        &ltdiv class="grid-container-3"&gt
+            &ltdiv class="row"&gt
+                &ltdiv class="col"&gt1 of 3&lt/div&gt
+                &ltdiv class="col"&gt2 of 3&lt/div&gt
+                &ltdiv class="col"&gt3 of 3&lt/div&gt   
+            &lt/div&gt
+        &lt/div&gt
+                </code>
+            </pre>
+        </div>
+        <!--Input-->
+        <h2 id="input">Inputs</h2>
+        <div class="div-example">
+            <div class="input">
+                <input type="text" class="field" placeholder="Name">
+            </div>
+            <div class="input">
+                <input type="text" class="field" placeholder="Email">
+            </div>
+        </div>
+
+        <h4>Code Snippets</h4>
+        <div class="code-snippet">
+            <pre>
+               <code>
+                &ltdiv class="input"&gt
+                    &ltinput type="text" class="field" placeholder="Name"&gt
+                &lt/div&gt
+                &ltdiv class="input"&gt
+                    &ltinput type="text" class="field" placeholder="Email"&gt
+                &lt/div&gt
+               </code>
+           </pre>
+        </div>
+        <!--Lists-->
+        <h2 id="list">Lists</h2>
+        <h4>Example</h4>
+        <div class="div-example div-lists">
+            <p>Stacked List</p>
+            <div>
+
+                <ul class="stacked-list">
+                    <li class="list-item">One</li>
+                    <li class="list-item">Two</li>
+                    <li class="list-item">Three</li>
+                    <li class="list-item">Four</li>
+                    <li class="list-item">Five</li>
+                </ul>
+            </div>
+            <p>Spaced List</p>
+
+            <div>
+                <ul class="spaced-list">
+                    <li class="list-item spaced">One</li>
+                    <li class="list-item spaced">Two</li>
+                    <li class="list-item spaced">Three</li>
+                    <li class="list-item spaced">Four</li>
+                    <li class="list-item spaced">Five</li>
+                </ul>
+            </div>
+        </div>
+        <h4>Code Snippet</h4>
```

Comment:
Code snippet text can be removed since it would be obvious by looking at the code snippet, or something like **Usage** can be used instead